json/json_test.go

```diff
@@ -1378,3 +1378,45 @@ func TestGithubIssue18(t *testing.T) {
 		}
 	}
 }
+
+func TestGithubIssue23(t *testing.T) {
+	t.Run("", func(t *testing.T) {
+		type d struct {
+			S map[string]string
+		}
+
+		b, _ := json.Marshal(map[string]d{
```

Comment:
This is using "encoding/json" & maybe other things are in these tests too that shouldn't be?

Reply:
Searching the file for "json." it looks like most uses are just as a reference/comparison ... maybe just `TestGithubIssue15` also has this mixup

Reply:
... and `TestDecodeStructFieldCaseInsensitive` ... that's probably it

Reply:
Thanks! I fixed all the misuses of the stdlib json package 👍